src/main/java/com/example/bankProjetc/services/impl/CustomerServiceImpl.java

```diff
@@ -52,9 +51,8 @@ public Customer updateCustomer(Customer customer, long id) {
 	}
 
 	@Override
-	public void deleteCustomer(long id) 
-	{
-		customerRepository.findById(id).orElseThrow(()->new ResourseNotFoundException("Customer", id));
+	public void deleteCustomer(long id) {
+		customerRepository.findById(id).orElseThrow(() -> new ResourseNotFoundException("Customer", id));
 		customerRepository.deleteById(id);
```

Comment:
don't delete from repo, add a field in entity for "is_cancelled" of type boolean and mark it true (called soft delete)